README.markdown

```diff
@@ -3138,6 +3138,7 @@ Nginx API for Lua
 * [ngx.req.append_body](#ngxreqappend_body)
 * [ngx.req.finish_body](#ngxreqfinish_body)
 * [ngx.req.socket](#ngxreqsocket)
+* [ngx.req.set_cong](#ngxreqset_cong)
```

Comment:
BTW, this file is automatically generated from `doc/HttpLuaModule.wiki`, to quote the comment at the beginning of this file:

```
cjson.safe: {"foo":123}
<!---
Don't edit this file manually! Instead you should generate it by using:
    wiki2markdown.pl doc/HttpLuaModule.wiki
-->
```